llvm-project/Makefile

```diff
@@ -0,0 +1,12 @@
+.DEFAULT_GOAL := clang
+
+%.svg: %.dot
+	dot -Tsvg $< > $@
+
+dotfiles: stage2.svg stage1.svg
+
+.PHONY: clang
+clang:
+	docker build --tag clangbuiltlinux/llvm-project:latest .
+	docker create --name llvm-project clangbuiltlinux/llvm-project:latest || true
```

Comment:
Maybe pull these out into their own variable (and add the container registry)?

```
TAG ?= ghcr.io/clangbuiltlinux/llvm-project:latest
...
        docker build --tag $(TAG) .
        docker create --name llvm-project $(TAG)
```

Reply:
I can dry this up; question about the tag name though.

I don't expect the Makefile to be used by the gh actions; only for local development.  In that case, shouldn't the tag be something (anything, nonsensical) that isn't our ghcr.io namespace, since the container will only be tagged/available locally?

Reply:
I see, thanks for the info.